charts/opentelemetry-operator/templates/certmanager.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ template "opentelemetry-operator.name" . }}-serving-cert
+  namespace: {{ template "opentelemetry-operator.namespace" . }}
+spec:
+  dnsNames:
+    - {{ template "opentelemetry-operator.name" . }}-webhook-service.{{ template "opentelemetry-operator.namespace" . }}.svc
+    - {{ template "opentelemetry-operator.name" . }}-webhook-service.{{ template "opentelemetry-operator.namespace" . }}.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: {{ template "opentelemetry-operator.name" . }}-selfsigned-issuer
```

Comment:
This should probably be something the user can specify in values and the creation of a self-signed issuer should also be optional.  If the user already has an issuer they want to use then they should be able to have this certificate created by that issuer.

Reply:
By this, do you mean that the user probably already has a `Issuer` or `ClusterIssuer` in the cluster. And they want to use that? Can you please elaborate on this?

Reply:
Yes, if a user has already configured an `Issuer` or `ClusterIssuer` with a different name then they should be able to 1) not have this helm chart create an `Issuer` and 2) use that `Issuer` or `ClusterIssuer` in this `Certificate`.

Reply:
Got it. This is more reasonable. I will make the changes.